EthereumKit/Helper/ERC20+Contract.swift

```diff
@@ -0,0 +1,26 @@
+extension ERC20 {
+    enum ContractFunctions {
+        case balanceOf(address: String)
+        
```

Comment:
Can you add Transfer case here please? its defined in ERC20 by `transferSigniture` property

Reply:
Refactored this enum to handle also the transfer method as well